service/iot-service-samples/device-twin-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/DeviceTwinSample.java

```diff
@@ -16,8 +16,8 @@
 /** Manages device twin operations on IotHub */
 public class DeviceTwinSample
 {
-    public static final String iotHubConnectionString = "[IOT HUB Connection String]";
-    public static final String deviceId = "[Device ID]";
+    private static final String iotHubConnectionString  = System.getenv("IOTHUB_CONNECTION_STRING");
```

Comment:
We should not accidentally check in these values so changed to using env variables